src/hardware/design_parameters.rs

```diff
@@ -17,6 +17,13 @@ pub const TIMER_PERIOD: f32 = 1. / (TIMER_FREQUENCY.to_Hz() as f32);
 /// The QSPI frequency for communicating with the pounder DDS.
 pub const POUNDER_QSPI_FREQUENCY: MegaHertz = MegaHertz::MHz(50);
 
+/// The QSPI frequency for LTC2320 readout.
+pub const DRIVER_QSPI_FREQUENCY: MegaHertz = MegaHertz::MHz(1); // Use slow 1 MHz for now. This will change.
```

Comment:
If you don't need it anywhere else, i.e. if it's not shared, don't put it into design parameters.